workflows/nf_aggregate/main.nf

```diff
@@ -34,15 +34,20 @@ workflow NF_AGGREGATE {
     )
     ch_versions = ch_versions.mix(SEQERA_RUNS_DUMP.out.versions.first())
 
+    ch_runs = SEQERA_RUNS_DUMP.out.run_dump
+        .branch {
+            gantt: it[0].fusion && ! params.skip_run_gantt
+            non_gantt: (! it[0].fusion) || (params.skip_run_gantt)
+        }
```

Comment:
```suggestion
    SEQERA_RUNS_DUMP
        .out
        .run_dump
        .branch {
            meta, run_dump ->
                gantt: meta.fusion && !params.skip_run_gantt
                no_gantt: !meta.fusion || params.skip_run_gantt
        }
        .set { ch_runs }
```

Reply:
Do we need to `branch` if we aren't using `no_gantt` anywhere? Could just use a `filter` instead.

Reply:
This was me thinking of things expanding in future, but OK, will switch to filter.